src/Component/AlbumBox/index.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+import AlbumCard from "../AlbumCard";
+
+function Card({ card }) {
+  return <AlbumCard src={card.src} />;
+}
+
```

Comment:
외부에서 `card` 라는 오브젝트를 가져오지만, 실제로 사용되고 있는 것은 `card.src` 같아요. 
외부에서 `card.src` 에 해당하는 값을 직접 넣어주면 아래와 같이 조금 더 개선해볼 수 있을 것 같군요!

```suggestion

function Card({ src }) {
  return <AlbumCard src={src} />;
}

```